src/core/xref.js

```diff
@@ -292,3 +296,17 @@ function disambiguate(fetchedData, context, term) {
   showInlineWarning(elem, msg, title);
   return null;
 }
+
+// utility functions for using `idb` as a key-value store
+function idbGet(key, db) {
```

Comment:
Can we get a class that restores some of the previous functions in `utils` so that we can use it from other modules e.g. `biblio-db`? Something like `export class IDBKeyVal { constructor(idb); get(); set();  }`.

Reply:
Will do!

Reply:
@saschanaz good now?